.github/workflows/build.yml

```diff
@@ -22,58 +22,10 @@ on:
 name: build
 
 jobs:
-  tests:
-    name: PHP ${{ matrix.php }}-${{ matrix.os }}
-
-    env:
-      key: cache-v1
-
-    runs-on: ${{ matrix.os }}
-
-    strategy:
-      matrix:
-        os:
-          - ubuntu-latest
-          - windows-latest
-
-        php:
-          - 7.4
-          - 8.0
-          - 8.1
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-
-      - name: Install PHP
-        uses: shivammathur/setup-php@v2
-        with:
-          php-version: ${{ matrix.php }}
-          ini-values: date.timezone='UTC'
-          coverage: pcov
-          tools: composer:v2
-
-      - name: Determine composer cache directory on Linux
-        if: matrix.os == 'ubuntu-latest'
-        run: echo "COMPOSER_CACHE_DIR=$(composer config cache-dir)" >> $GITHUB_ENV
-
-      - name: Determine composer cache directory on Windows
-        if: matrix.os == 'windows-latest'
-        run: echo "COMPOSER_CACHE_DIR=~\AppData\Local\Composer" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
-
-      - name: Cache dependencies installed with composer
-        uses: actions/cache@v3
-        with:
-          path: ${{ env.COMPOSER_CACHE_DIR }}
-          key: php${{ matrix.php }}-composer-${{ hashFiles('**/composer.json') }}
-          restore-keys: |
-            php${{ matrix.php }}-composer-
-
-      - name: Update composer
-        run: composer self-update
-
-      - name: Install dependencies with composer
-        run: composer update --prefer-dist --no-interaction --no-progress --optimize-autoloader --ansi
-
-      - name: Run tests with phpunit
-        run: vendor/bin/phpunit --colors=always
+  phpunit:
+    uses: yiisoft/actions/.github/workflows/phpunit.yml@master
```

Comment:
In `yiisoft/actions` not need run actions. Suggest move them to root (`yiisoft/actions/phpunit.yml`)

Reply:
Please read the documentation.

Reply:
OK. https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_iduses